server/data_access_layer/daos/transac_stat.dao.js

```diff
@@ -37,21 +37,21 @@ exports.getTransactionsStatByYearMonth = async (yearMonthList, employeeId = unde
     })
 }
 
-exports.prepareDuesQuery = (yearMonthList, employeeId, clientType, countryLabel, ageOfAccount) => {
+exports.prepareDuesQuery = (yearMonthList, employeeId, clientType, countryLabel, ageOfAccount, accountType) => {
     let query = {
         queryString: "SELECT ACS.DUE_CURRENT, ACS.DUE_1_MONTH, ACS.DUE_2_MONTH, ACS.DUE_3_MONTH, ACS.YEAR_MONTH ",
         replacements: [yearMonthList]
     };
 
     let fromString = "FROM ACCOUNTING_CLIENT_STAT ACS ";
-    let whereString = "WHERE ACS.YEAR_MONTH in (?) AND ACS.CONNECTION_ID=3 AND ACS.STAT_TYPE=1";
+    let whereString = "WHERE ACS.YEAR_MONTH in (?) AND ACS.STAT_TYPE=1";
```

Comment:
Why did you removed the ACS.CONNECTION_ID = 3 here?

Reply:
It is below in the account type condition now since we need to swithc between the two